### ISSUE TITLE
Add back about page 

### ISSUE THREAD

**Issue by SkyMTF** [mastodon/mastodon#20798]
### Pitch

Add back the about/landing page of the mastodon instance 

### Motivation

Removing the about page makes it harder for users to learn what an instance is about, and makes it harder for rules to be clearly stated before signing up.

**Comment by ClearlyClaire** [comment]
The new `/about` page does communicate all of this. However, it is not the landing page when trends are enabled, which is by design but sounds wrong to me. I'd like there to be a setting for `/about` to be the landing page no matter what, at the very least.

**Comment by aphyr** [comment]
I'd like to second this. On woof.group we had extensive new-user documentation and policy guidance on `/about/more`. In 4.0 `/about/more` for logged in users appears to redirect to `/about`, which in turn opens the regular Mastodon web interface rather than showing the about page.

Is it even *possible* for users to see their instance policies without an incognito window any more?

**Comment by ClearlyClaire** [comment]
> I'd like there to be a setting for `/about` to be the landing page no matter what, at the very least.

https://github.com/mastodon/mastodon/pull/20808

> Is it even _possible_ for users to see their instance policies without an incognito window any more?

Yes, it's the “About” link in the footer besides your instance's name.

**Comment by aphyr** [comment]
Ahhhhh it loads in dynamically that little pane on the far right, and I guess some sort of JS thing broke the first few times I tried it so it didn't load right.

Even when it does load this is... not a great reading experience. I'd really like to have a full-width dedicated page for this kind of thing.

![Screenshot from 2022-11-15 13-18-38](https://user-images.githubusercontent.com/3748/201995909-0c8b04f6-d863-4ddb-bd84-8df0fc67ab51.png)

**Comment by egypturnash** [comment]
@ClearlyClaire *I'd like there to be a setting for /about to be the landing page no matter what, at the very least.*

very yes

very very yes

also I would like some guarantee that /about will not get completely rewritten every major release, I spent a lot of time customizing that back in 2017 and I have had to swear at webpack so damn much every time I update; it's been a broken mess since 3.4.6 because I haven't had the energy or time to fix it since having to update for whatever major security hole that was fixing.

**Comment by kococomi** [comment]
Admins should be allowed to change their landing page easily like what Pleroma does.

<img width="874" alt="Screen Shot 2022-11-16 at 6 32 55 am" src="https://user-images.githubusercontent.com/76918202/202009167-414f1c01-1d9d-4066-a36e-111d822072b4.png">


To me the `/about` page requires a different UI design, the 3-panel layout looks so weird as a majority part of the main panel has information that is exactly the same as the left one. This is a waste of space...

<img width="1377" alt="Screen Shot 2022-11-16 at 6 27 50 am" src="https://user-images.githubusercontent.com/76918202/202009671-b4d1cb30-1c38-4d5b-878c-0fbcd96ffb75.png">

**Comment by psydwannabe** [comment]
I think we can do better than the /about page, which was very limited in its utility and customizability. I proposed instead we push for a customizable homepage:
https://github.com/mastodon/mastodon/issues/20786 

The proposal from @kococomi above to include a landing page redirection URL is a great suggestion too; that makes it easy to run a full CMS site to support the instance app, instead of trying to shoehorn a CMS into the app.

I get the impression (from the switch from HTML to Markdown Only in server descriptions) that @Gargron is focused on turning our unique communities into a unified brandless Mastodon experience that can be more readily packaged and sold as a Twitter replacement, in which case these sorts of requests will almost assuredly fall on deaf and uncaring ears.

**Comment by ryliejamesthomas** [comment]
I do like that the rules (etc.) are now on the same page as the signup form, hopefully makes it more likely people will actually read them before applying.

**Comment by joenepraat** [comment]
See also this issue: https://github.com/mastodon/mastodon/issues/20930, what is a major security flaw. How does not anybody raised a red flag during development!?

**Comment by pullopen** [comment]
As an instance owner, I also don't want any unauthorized visit to the "explore" page. I want to show a mere "about" page to unlogged users for security reason. I hope it can be done in the future version.

**Comment by ryliejamesthomas** [comment]
> As an instance owner, I also don't want any unauthorized visit to the "explore" page. I want to show a mere "about" page to unlogged users for security reason. I hope it can be done in the future version.

Not a solution, but a workaround: I removed the explore page by disabling trend stuff.

**Comment by pullopen** [comment]
> > As an instance owner, I_ also _don't want any unauthorized visit to the "explore" page. I want to show a mere "about" page to unlogged users for security reason. I hope it can be done in the future version.
> 
> Not a solution,_ but _a workaround: I removed the explore page by disabling trend stuff.

Yeah, but still sometimes I found the explore feature useful. So I put off the upgrading lol

**Comment by vmstan** [comment]
About page option was added back to Mastodon 4.1 by #20808
